DTML.EduBot/DTML.EduBot.csproj

```diff
@@ -157,6 +157,7 @@
     <Compile Include="LessonPlan\LessonPlanModule.cs" />
     <Compile Include="LessonPlan\Lesson.cs" />
     <Compile Include="LessonPlan\LessonPlan.cs" />
+    <Compile Include="LessonPlan\TestQuestion.cs" />
```

Comment:
The tests should be separate from main code. I think it is ok for now but overall we need to keep separation.